packages/generator/src/react/context.ts

```diff
@@ -15,9 +15,28 @@ export class DefaultImportBuilder {
         //formattedDateTimeRange.importDeclaration, formattedRelativeTime.importDeclaration, 
         //formattedPlural.importDeclaration, formattedMessage.importDeclaration
     }
-
-    unique() {
-        return this.imports // TODO unique
+    
+    unique() {     
```

Comment:
This functionality will work on all types of symbol imports as you want. If we want it to work on duplicate import names or namespace imports it have to be modified.